lib/sync/init.go

```diff
@@ -6,7 +6,7 @@ import (
 	"boscoin.io/sebak/lib/common"
 )
 
-var log logging.Logger = logging.New("module", "sync")
+var log logging.Logger = logging.New()
```

Comment:
By default, leaving the `module` will be better.

Reply:
I changed it! @spikeekips  :->